packages/context/src/authentication/AuthenticationContextProvider.tsx

```diff
@@ -3,14 +3,16 @@
 
 import { AccountInfo, RedirectRequest } from '@azure/msal-browser';
 import { IMsalContext } from '@azure/msal-react';
+import { FusionAuthProviderContext } from '@fusionauth/react-sdk';
 import { ExtraSigninRequestArgs } from 'oidc-client-ts';
 import { ReactNode, createContext, useCallback, useContext, useEffect, useMemo, useRef } from 'react';
 import { AuthContextProps } from 'react-oidc-context';
 
 export type AuthenticationContext = {
     account: UserAccount;
     logout: VoidFunction;
-    getAccessToken: () => Promise<string>;
+    getAccessToken?: () => Promise<string>;
```

Comment:
Making this optional is a breaking change

Reply:
It's a breaking change, meaning any custom widgets calling `getAccessToken` would break.